stdlib/public/core/VarArgs.swift

```diff
@@ -498,6 +515,129 @@ final internal class _VaListBuilder {
                Builtin.addressof(&self.header)))
   }
 }
+#elseif arch(arm64) && os(Linux)
```

Comment:
Similar to the other cases for the enabling.

Reply:
As answered above.